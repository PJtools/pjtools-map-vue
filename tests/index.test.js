import pkg from '../package.json';

const testDist = process.env.LIB_DIR === 'dist';

describe('pjmap dist files', () => {
  it('exports modules correctly', () => {
    const pjmap = testDist ? require('../dist/pjmap') : require('../components'); // eslint-disable-line global-require
    expect(Object.keys(pjmap)).toMatchSnapshot();
  });

  if (testDist) {
    it('should have pjmap.version', () => {
      const pjmap = require('../dist/pjmap'); // eslint-disable-line global-require
      expect(pjmap.version).toBe(pkg.version);
    });
  }
});
