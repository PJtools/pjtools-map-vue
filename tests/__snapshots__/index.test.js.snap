// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pjmap dist files exports modules correctly 1`] = `
Array [
  "Base",
  "Map",
  "version",
  "install",
  "preload",
  "default",
]
`;
